src/ts/components/core/Burger.tsx

```diff
@@ -17,6 +17,8 @@ interface Props
     size?: MantineSize | (string & {}) | number;
     /** Key of `theme.colors` of any valid CSS value, by default `theme.white` in dark color scheme and `theme.black` in light */
     color?: MantineColor;
+    /** Height of the burger lines */
+    lineSize?: string | number;
```

Comment:
As @AnnMarieW commented:

> Based on the [Mantine Docs](https://mantine.dev/core/burger/#change-lines-size), the `lineSize` prop is a number.  For example `lineSize=5`  or as text `lineSize="5"`   but the Mantine named sizes like `xl` is not valid

If named sizes like `xl` are in fact valid, let's put one in the test. If not, let's remove `string |` here.

Reply:
It doesn't work with the named sizes.  I tried it in a code sandbox with the Mantine react component too, and it can either be `lineSize={5}` or `lineSize="5"` but nothing renders if you do `lineSize="sm"`

@ChinoUkaegbu  would you like to update to the prop so it's just 
```
 lineSize?: number;
```

Reply:
Sure thing!